backend/src/main/java/com/exadel/carinsurance/mapper/AssignmentMapper.java

```diff
@@ -0,0 +1,71 @@
+package com.exadel.carinsurance.mapper;
+
+import com.exadel.carinsurance.model.assignment.AssignmentEntity;
+import com.exadel.carinsurance.model.assignment.ContactInfoEntity;
+import com.exadel.carinsurance.model.request.AssignmentRequestEntity;
+import com.exadel.carinsurance.model.response.AssignmentResponseEntity;
+import com.exadel.carinsurance.model.response.ContactInfoResponseEntity;
+import com.exadel.carinsurance.model.response.DirectionOfImpactResponseEntity;
+import com.exadel.carinsurance.model.response.VehicleConditionInfoResponseEntity;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Component
+public class AssignmentMapper implements IMapper<AssignmentEntity, AssignmentRequestEntity, AssignmentResponseEntity> {

```

Comment:
it is better to have mappers for 2 entries not 3.
Think of it as a single responsibility mapper which purpose is to convert object from <source> to <target> and vice versa

Reply:
Done